pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -270,6 +270,19 @@ func (key *ChunkCheckpointKey) less(other *ChunkCheckpointKey) bool {
 	}
 }
 
+// Chunk records the chunk information.
+type Chunk struct {
```

Comment:
please move to `importer` pkg. this `checkpoints` pkg is mostly specific to lightning

Reply:
Modified it.